### ISSUE TITLE
Dark icon

### ISSUE THREAD

**Issue by happysurf** [mozilla/FirefoxColor#846]
In the Firefox addons dark page, the FirefoxColor icon is hard to see, will be great make a medium gray or something like that.

![Dark Icon2](https://user-images.githubusercontent.com/13637587/66698466-af552b00-ecde-11e9-99a2-421b9ceeb3f1.png)

**Comment by ShivamRawat0l** [comment]
Can i work on this issue ?

**Comment by happysurf** [comment]
For apply the dark theme set **browser.in-content.dark-mode** to **true** (default in Firefox 70 beta).

**Comment by ShivamRawat0l** [comment]
Does this fix the issue ?

**Comment by happysurf** [comment]
> 
> Does this fix the issue ?

I think so if the color is #686868.

**Comment by Rob--W** [comment]
I wonder whether we should add a contrasting color (i.e. a light color) around the black Firefox Color logo to make the icon dark theme-friendly.

**Comment by happysurf** [comment]
Same problem with other Mozilla add-on Side View, i think the best choice is follow the plugin page icon colors for activated / disactivated state.

![Icons](https://user-images.githubusercontent.com/13637587/66893983-a2427f80-efef-11e9-92bd-ec954e7f29cf.png)

**Comment by ShivamRawat0l** [comment]
@Rob--W  @happysurf  Yes we could change the logo. Or we can also add support for theme icons in the firefox addon section.
